Week1/exercise-2/exercise2.js

```diff
@@ -0,0 +1,72 @@
+const mysql = require("mysql")
+
+
+var con = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    database: 'world'
+});
+
+con.connect();
+
+con.query('SELECT name FROM country WHERE Population > 8000000', function (err, result) {
+    if (err) throw err;
+    console.log(`Countries with population over 8 million are:`, result);
+});
+
+con.query('SELECT NAME FROM COUNTRY WHERE NAME LIKE "%LAND%";', function (err, result) {
+    if (err) throw err;
+    console.log(`Countries with LAND in their name:`, result);
+});
+
+con.query(' SELECT name FROM country WHERE population BETWEEN 500000 AND 1000000;', function (err, result) {
+    if (err) throw err;
+    console.log(`Countries with POPULATION between 500000 and 1000000:`, result);
+});
+
+con.query(' SELECT name FROM country WHERE population BETWEEN 500000 AND 1000000;', function (err, result) {
+    if (err) throw err;
+    console.log(`Countries with POPULATION between 500000 and 1000000:`, result);
+});
+
+con.query('SELECT name FROM country WHERE continent LIKE "Europe";', function (err, result) {
```

Comment:
Unnecessary `LIKE`. Should've been an equality check instead (`= "Europe"`). LIKE is slower.